actors/builtin/multisig/multisig_actor.go

```diff
@@ -423,21 +417,35 @@ func (a Actor) approveTransaction(rt vmr.Runtime, txnID TxnID, proposalHash []by
 }
 
 func isSigner(rt vmr.Runtime, st *State, address addr.Address) bool {
+	candidateIdAddr := address
```

Comment:
It won't necessarily resolve to an ID address, so `candidateResolved` would be slightly more accurate.